streaming/aggregate/last.go

```diff
@@ -0,0 +1,110 @@
+package aggregate
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+
+	"github.com/cube2222/octosql"
+	"github.com/cube2222/octosql/streaming/storage"
+)
+
+var currentLastPrefix = []byte("$current_last$")
+var currentLastCountsPrefix = []byte("$current_last_counts$")
+
+type Last struct {
+}
+
+func NewLastAggregate() *Last {
+	return &Last{}
+}
+
+func (agg *Last) AddValue(ctx context.Context, tx storage.StateTransaction, value octosql.Value) error {
```

Comment:
Same comment applies as in first - if you put a comment here, reading the code the first time would be a bit easier :)